.nycrc

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
is this javascript or JSON?
could we add `/* this is file for create instrumented version of module.js to allow get coverage */`

Reply:
JSON, I can add comment as part of json: `"_comment": "this is file for create instrumented version of module.js to allow get coverage",`. Shall I?

Reply:
yep, please add it